packages/i18nify-js/README.md

```diff
@@ -540,3 +540,50 @@ console.log(getDialCodes()); /*
 console.log(getDialCodeByCountryCode('BR')); // Outputs the dial code for Brazil (+55)
 console.log(getDialCodeByCountryCode('DE')); // Outputs the dial code for Germany (+49)
 ```
+
+### Module 03: Geo Module 🌍
+
+Dive into the digital atlas with the Geo Module 🌍, your go-to source for fetching flag emojis 🏁 and images from around the globe 🌐. Whether you're jazzing up your app with patriotic flair 🎉 or feeding your curiosity about different countries 🤔, this module is your magic carpet ride 🧞‍♂️. Featuring two fantastic functions ✨, it's never been easier to bring a world of flags 🚩 right into your app. Let's unpack these global treasures 🌟:
+
+#### getFlagOfCountry(countryCode) 🏁
+
```

Comment:
Include a link for https://flagcdn.com/ and highlight that we are using this open source service for flags. 
Let's appreciate this service too.